src/test/java/com/ericsson/bss/cassandra/ecaudit/auth/TestAuditWhitelistManager.java

```diff
@@ -206,20 +146,20 @@ public void testRevokeAtAlter()
     {
         when(performer.getPermissions(any())).thenReturn(ImmutableSet.of(Permission.AUTHORIZE));
         RoleOptions options = createRoleOptions(
-                Collections.singletonMap("revoke_audit_whitelist_for_all", "connections"));
+                Collections.singletonMap("revoke_audit_whitelist_for_execute", "connections"));
 
         whitelistManager.alterRoleWhitelist(performer, role, options);
 
-        verify(mockWhitelistDataAccess, times(1)).removeFromWhitelist(eq(role), eq("ALL"),
-                eq(ImmutableSet.of(ConnectionResource.fromName("connections"))));
+        verify(mockWhitelistDataAccess, times(1))
+        .removeFromWhitelist(eq(role), eq(ConnectionResource.fromName("connections")), eq(ImmutableSet.of(Permission.EXECUTE)));
     }
 
```

Comment:
MIssing test cases
- Reject several grants in one operation
- Reject several resources in one grant operation.